DataSynapseApi/ArmoniK.DevelopmentKit.GridServer.Client/Properties.cs

```diff
@@ -73,19 +74,19 @@ public class Properties
     public Properties(TaskOptions options,
                       string      connectionAddress,
                       int         connectionPort = 0,
-                      string      protocol       = null,
-                      string      clientCertPem  = null,
-                      string      clientKeyPem   = null,
-                      string      caCertPem      = null,
-                      bool        sslValidation  = true) : this(new ConfigurationBuilder().Build(),
-                                                                options,
-                                                                connectionAddress,
-                                                                connectionPort,
-                                                                protocol,
-                                                                clientCertPem,
-                                                                clientKeyPem,
-                                                                caCertPem,
-                                                                sslValidation)
+                      string      protocol = null,
+                      string      clientCertPem = null,
+                      string      clientKeyPem = null,
+                      string      caCertPem = null,
+                      bool        sslValidation = true) : this(new ConfigurationBuilder().SetBasePath(Directory.GetCurrentDirectory()).AddEnvironmentVariables().Build(),
```

Comment:
Why do you need to get the Path of the current directory to get the environment variable?

Reply:
To find file in the workingdirectory. A relative path in configuration could have to search from the currentDirectory called by Users

Reply:
I do not understand why you need to search in the current directory if you are adding the environment variables.